example/demo/src/app.rs

```diff
@@ -71,13 +71,13 @@ fn HomePage() -> impl IntoView {
     );
 
     // scope_a related hotkeys
-    use_hotkeys!(("arrowup", "scope_a") => move |_| {
+    use_hotkeys!(("arrowup,arrowright", "scope_a") => move |_| {
```

Comment:
btw @JustBobinAround we can bind multiple key combos with the `,` sym.

This means `ctrl+k,arrowup` means ctrl+k and uparrow both trigger the same callback

Reply:
I thought this happens already:
```rust
pub fn use_hotkeys_ref_scoped<T>( 
    key_combination: String,
    on_triggered: Callback<()>,
    scopes: Vec<String>,
) -> NodeRef<T>
where
    T: ElementDescriptor + 'static + Clone,
{
    let node_ref = create_node_ref::<T>();

    create_effect(move |_| {
        let parsed_keys: HashSet<Hotkey> = key_combination
            .split(',') // <-- Right here and is the same for use_hotkeys_scoped
            .map(|key_combo| parse_key(key_combo))
            .collect();
```
Maybe I'm misunderstanding?

Reply:
Nope! You're correct. Just wanted to state because none of the hotkeys in the demo do multiple

Reply:
I think he just wants to trim whitespace, so just add a trim() after split